flask_admin/contrib/sqla/view.py

```diff
@@ -611,6 +614,27 @@ def scaffold_form(self):
 
         return form_class
 
+    def scaffold_list_form(self):
+        """
+            Create form for the list view editable columns.
+            
+            The form is created using the existing get_form(),
+            but each field is wrapped in a WTForms FieldList. 
+        """
+        converter = self.model_form_converter(self.session, self)
+        form_class = form.get_form(self.model, converter,
+                                   base_class=self.form_base_class,
+                                   only=self.column_editable_list)
+        
+        # iterate FormMeta to get unbound fields
+        field_dict = {}
```

Comment:
Here's the helper function I created: https://github.com/mrjoes/flask-admin/pull/756/files#diff-e1ef986502cb727b91e5215c566f57c6R17

I noticed `scaffold_list_forms` needs to be implemented in each backend's view.py because get_form is implemented differently for each.